src/components/AutoComplete/AutoComplete.component.tsx

```diff
@@ -0,0 +1,178 @@
+/* eslint-disable @typescript-eslint/indent */
+import React, {useCallback, useEffect, useRef, useState} from 'react';
+import _, {debounce} from 'lodash';
+
+import {Input} from '@Components/Input';
+import {ListItemType} from '@Components/List/List';
+
+import {AutoCompleteProps} from './AutoComplete';
+import {AutoCompleteWrapper} from './AutoComplete.styles';
+import {List} from '../List/List.component';
```

Comment:
can't this be imported as `@Components/List/List.component`? just like you did on line 5-6